server/app_env/_base.py

```diff
@@ -91,3 +91,6 @@ class Config:
   MIN_STAT_VAR_GEO_COVERAGE = 10
   # NL Bad words file.
   BAD_WORDS_FILE = 'nl_bad_words.txt'
+  # Whether to enable BigQuery from instance. This is primarily used for
```

Comment:
nit: The comment does not look right. Can you correct it?

Reply:
Done